modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/shared/ClaySelect.es.js

```diff
@@ -0,0 +1,40 @@
+import React, {Component} from 'react';
+import {PropTypes} from 'prop-types';
+import getCN from 'classnames';
+
+class ClaySelect extends Component {
+	static propTypes = {
+		onChange: PropTypes.func.isRequired,
+		options: PropTypes.array.isRequired,
+		selected: PropTypes.string
+	};
+
+	render() {
+		const {className, options, selected, ...otherProps} = this.props;
+
+		const classes = getCN(
+			'form-control',
+			{
+				[className]: className
```

Comment:
I think you can just do `getCN('form-control', className)` right?

Reply:
Oh right. That does work. 👍